js/bitopro.js

```diff
@@ -0,0 +1,1151 @@
+        //             "action":"SELL",
+        //             "type":"LIMIT",
+        //             "timestamp":1644899002598,
+        //             "status":4,
+        //             "originalAmount":"0.01",
+        //             "remainingAmount":"0.01",
+        //             "executedAmount":"0",
+        //             "fee":"0",
+        //             "feeSymbol":"twd",
+        //             "bitoFee":"0",
+        //             "total":"0",
+        //             "seq":"BNBTWD548774666",
+        //             "timeInForce":"GTC",
+        //             "createdTimestamp":1644898944074,
+        //             "updatedTimestamp":1644899002598
+        //         }
+        //
+        let id = this.safeString (order, 'orderId');
+        if (id === undefined) {
+            id = this.safeString (order, 'id');
+        }
+        const timestamp = this.safeInteger (order, 'timestamp');
+        const side = this.safeString (order, 'action');
+        const amount = this.safeString (order, 'amount');
+        const price = this.safeString (order, 'price');
+        const marketId = this.safeString (order, 'pair');
+        if (marketId !== undefined) {
+            market = this.market (marketId);
+        }
+        const symbol = this.safeString (market, 'symbol');
+        const orderStatus = this.safeString (order, 'status');
+        const status = this.parseOrderStatus (orderStatus);
+        const type = this.safeString (order, 'type');
+        const average = this.safeString (order, 'avgExecutionPrice');
+        const filled = this.safeString (order, 'executedAmount');
+        const remaining = this.safeString (order, 'remainingAmount');
+        const timeInForce = this.safeString (order, 'timeInForce');
+        let fee = undefined;
+        const feeAmount = this.safeString (order, 'fee');
+        const feeSymbol = this.safeString (order, 'feeSymbol');
+        if (feeAmount !== undefined) {
+            fee = {
+                'currency': feeSymbol,
+                'cost': feeAmount,
+            };
+        }
+        return this.safeOrder ({
+            'id': id,
+            'clientOrderId': undefined,
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'lastTradeTimestamp': undefined,
+            'symbol': symbol,
+            'type': type,
+            'timeInForce': timeInForce,
+            'postOnly': undefined,
+            'side': side,
+            'price': price,
+            'stopPrice': undefined,
+            'amount': amount,
+            'cost': undefined,
+            'average': average,
+            'filled': filled,
+            'remaining': remaining,
+            'status': status,
+            'fee': fee,
+            'trades': undefined,
+            'info': order,
+        }, market);
+    }
+
+    async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        await this.loadMarkets ();
```

Comment:
We should support stopOrders (STOP_LIMIT in this case) too. Also, take into consideration that if the user passes the "type" "limit" but also adds a "stopPrice" to parameters, this should be encoded as a stop_Limit order

![image](https://user-images.githubusercontent.com/43336371/154240234-129c75b9-d867-45d5-8f16-acc46c746956.png)